### ISSUE TITLE
dvclive: Add a note _not_ to use `dvclive.init` when using keras Callback

### ISSUE THREAD

**Issue by iesahin** [iterative/dvc.org#2773]
I think it's better to notify the user that `dvclive.init` is called inside the callback in https://dvc.org/doc/dvclive/user-guide/ml-frameworks/keras 

They also don't need ìmport dvclive`. 

@daavoo @dberenbaum

**Comment by dberenbaum** [comment]
I'm not sure I understand. This example is meant to show everything that needs to be added to integrate dvclive into an existing keras script, and it doesn't include `import dvclive` or `dvclive.init`. Do you think we need to clarify that no other `dvclive` code is needed?

**Comment by iesahin** [comment]
Yes, that's what I mean. In the [docs](https://dvc.org/doc/dvclive/api-reference/init) we say `init` is optional, but from the errors I got in the new project, it seems like an error to use `dvclive.init` when using `DvcLiveCallback`.

**Comment by dberenbaum** [comment]
Good catch! Maybe this is a bug in dvclive? I would think it should be possible to use `init` with the callbacks, or else it limits the flexiblity to use different paths and other options. @daavoo What do you think?

**Comment by daavoo** [comment]
> Good catch! Maybe this is a bug in dvclive? I would think it should be possible to use `init` with the callbacks, or else it limits the flexiblity to use different paths and other options. @daavoo What do you think?

I think that in frameworks for which we provide an integration callback users should not call `dvclive.init`. 

We should be actually calling `dvclive.init` ourselves inside those integrations (currently we don't) and expose the `init` args as part of the callback args.
